pkg/cloudprovider/providers/azure/azure_instances.go

```diff
@@ -14,21 +14,22 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package azure
+package main: azure
```

Comment:
I'm actually not sure what this does, but I don't think we should be doing it :-)  It appears to stop the code from compiling